### ISSUE TITLE
Heapster runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by Galexrt** [kubernetes/heapster#735]
### Description
Heapster is periodically crashing like every 6 hours.

***

### Details
* **OS**: Fedora release 22 (Twenty Two)
* **Versions**:
    * **Docker**:
```
Client:
 Version:      1.8.2-fc22
 API version:  1.20
 Package Version: docker-1.8.2-7.gitcb216be.fc22.x86_64
 Go version:   go1.5.1
 Git commit:   cb216be/1.8.2
 Built:        
 OS/Arch:      linux/amd64

Server:
 Version:      1.8.2-fc22
 API version:  1.20
 Package Version: 
 Go version:   go1.5.1
 Git commit:   cb216be/1.8.2
 Built:        
 OS/Arch:      linux/amd64
```
* **Kubernetes**:
```
kubectl version
Client Version: version.Info{Major:"1", Minor:"1", GitVersion:"v1.1.2", GitCommit:"3085895b8a70a3d985e9320a098e74f545546171", GitTreeState:"clean"}
Server Version: version.Info{Major:"1", Minor:"1", GitVersion:"v1.1.2", GitCommit:"3085895b8a70a3d985e9320a098e74f545546171", GitTreeState:"clean"}
```
* **Heapster**: `gcr.io/google_containers/heapster:v0.18.2`

****

### Output
**Kubernetes addon file** `/etc/kubernetes/addons/cluster-monitoring/heapster-controller.yaml`:
```
apiVersion: v1
kind: ReplicationController
metadata:
  name: heapster-v10
  namespace: kube-system
  labels:
    k8s-app: heapster
    version: v10
    kubernetes.io/cluster-service: "true"
spec:
  replicas: 1
  selector:
    k8s-app: heapster
    version: v10
  template:
    metadata:
      labels:
        k8s-app: heapster
        version: v10
        kubernetes.io/cluster-service: "true"
    spec:
      containers:
        - image: gcr.io/google_containers/heapster:v0.18.2
          name: heapster
          resources:
            limits:
              cpu: 100m
              memory: 768Mi
          command:
            - /heapster
            - --source=kubernetes:''
            - --sink=influxdb:http://monitoring-influxdb:8086
            - --stats_resolution=30s
            - --sink_frequency=1m
```

**Heapster log outpu**:
https://gist.github.com/Galexrt/cb01deeb754e43963091

**Comment by jimmidyson** [comment]
Thanks for reporting.

This is a dupe of #658 fixed in master by #693

**Comment by Galexrt** [comment]
@jimmidyson Sry for the dupe, I didn't saw those issues.
